### ISSUE TITLE
Include grpc-web dependencie

### ISSUE THREAD

**Issue by quetool** [grpc/grpc-dart#171]
Hi, wich is the correct way to include grpc-web in an AngularDart proyect?
Here is my pubspec
```
name: adminpanelv3
description: An absolute bare-bones web app.
# version: 1.0.0
#homepage: https://www.example.com
#author: alfredo <email@example.com>

environment:
  # sdk: ">=2.1.0 <3.0.0"
  sdk: ">=2.2.1-dev.3.0 <3.0.0"

dependencies:
  angular: ^5.3.0
  angular_components: ^0.13.0
  http: ^0.12.0+2
  grpc:
    git:
      url: https://github.com/grpc/grpc-dart
      ref: grpc-web
      resolved-ref: af63deb

  protobuf: ^0.13.11

dev_dependencies:
  build_runner: ^1.3.3
  build_web_compilers: ^2.0.0
  pageloader: ^3.2.0
```

I am getting a lot of errors
```
AlfredoMBPro:adminweb alfredo$ webdev serve
[INFO] Serving `web` on http://localhost:8080
[WARNING] build_web_compilers|ddc_modules on package:protos/.ddc.meta_module.raw: Unable to read module information for package:grpc, make sure you have a dependency on it in your pubspec.
[SEVERE] build_web_compilers|ddc on package:protos/admin.ddc.module: Unable to find modules for some sources, this is usually the result of either a
bad import, a missing dependency in a package (or possibly a dev_dependency
needs to move to a real dependency), or a build failure (if importing a
generated file).

Please check the following imports:


[SEVERE] build_web_compilers|ddc on package:protos/admin.ddc.module: Unable to find modules for some sources, this is usually the result of either a
bad import, a missing dependency in a package (or possibly a dev_dependency
needs to move to a real dependency), or a build failure (if importing a
generated file).

Please check the following imports:

`import 'package:grpc/service_api.dart' as $grpc;` from protos|lib/src/admin.pbgrpc.dart at 9:1

[SEVERE] build_web_compilers|ddc on lib/app_component.template.ddc.module: 
[SEVERE] build_web_compilers|ddc on web/main.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[WARNING] build_web_compilers|entrypoint on web/main.dart: Unable to read protos|lib/admin.ddc.js, check your console or the `.dart_tool/build/generated/protos/lib/admin.ddc.js.errors` log file.
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.ddc.module: 
[WARNING] build_web_compilers|entrypoint on web/main.dart: Unable to read adminpanelv3|lib/app_component.ddc.js, check your console or the `.dart_tool/build/generated/adminpanelv3/lib/app_component.ddc.js.errors` log file.
[SEVERE] build_web_compilers|ddc on lib/app_component.template.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.template.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.template.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.template.ddc.module: 
[SEVERE] build_web_compilers|ddc on lib/app_component.template.ddc.module: 
[WARNING] No actions completed for 15.0s, waiting on:
  - build_web_compilers|ddc on lib/app_component.ddc.module
  - build_web_compilers|ddc on lib/app_component.template.ddc.module
  - build_web_compilers|entrypoint on web/main.dart
```

**Comment by sigurdm** [comment]
```
  grpc:
    git:
      url: https://github.com/grpc/grpc-dart
      ref: grpc-web
      resolved-ref: af63deb
```
Seems right, and I'm able to use it locally.

Not sure what goes wrong...

Stupid question: did you remember to do `pub get`?

**Comment by varpon** [comment]
This is working for me when using an older version of build_web_compiler (<2.0.0).
